### PR TITLE
small clean up: var,let,const ;

### PR DESCRIPTION
**Pull request by Zwilla**
make use of let and const instead of var.

### CODE REVIEW

**Review by Zwilla — COMMENTED** [review]
now it must work. I tried to implement a no cache function. Later more.

File: gulpfile.js [hunk 1]
```diff
@@ -132,26 +132,26 @@ function bundle_js_debug(bundler) {
...
 
 
 
 // Rebuild Static JS
-var js_srcFilesStatic = app + 'scripts/staticJS/to-compile-to-static/*.js'
-var js_destFolderStatic = app + 'scripts/staticJS/'
-var js_destFileStatic = 'etherwallet-static.min.js'
+let js_srcFilesStatic = app + 'scripts/staticJS/to-compile-to-static/*.js';
+let js_destFolderStatic = app + 'scripts/staticJS/';
+let js_destFileStatic = 'tokenwallet-static.min.js';
```
Comment (crptm) [inline gulpfile.js]: ?

Comment (Zwilla) [reply gulpfile.js]: thx, forgot to change!

**Review by Zwilla — COMMENTED** [review]
now it can pass

**Pull request merged**